src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -76,8 +86,15 @@ public async Task<IActionResult> Index(PagerParameters pagerParameters, bool adm
             var count = templatesDocument.Templates.Count;
 
             var templates = templatesDocument.Templates.OrderBy(x => x.Key)
-                .Skip(pager.GetStartIndex())
-                .Take(pager.PageSize);
+                            .Skip(pager.GetStartIndex())
+                            .Take(pager.PageSize);
```

Comment:
Ezt a szűrést akkor is lefuttatod, ha van `displayText`, de nem kellene.

Reply:
Létre kéne hozni a templates változót if -en kívül hogy máshol is használható legyen, de csak úgy tudok neki értéket adni ha már eleve azt mondom hogy templatesDocument.Templates.Take()

Reply:
Változót értékadás nélkül is létre tudsz hozni.

Reply:
Elvileg megvan, majd megnézzük hogy így gondoltad e.